node-commons/src/main/java/org/apache/iotdb/commons/auth/user/LocalFileUserAccessor.java

```diff
@@ -98,9 +98,9 @@ public User loadUser(String username) throws IOException {
         return null;
       }
     }
-    try (FileInputStream inputStream = new FileInputStream(userProfile);
-        DataInputStream dataInputStream =
-            new DataInputStream(new BufferedInputStream(inputStream))) {
+    FileInputStream inputStream = new FileInputStream(userProfile);
```

Comment:
Why move the code out of try?

Reply:
Because FileInputStream will be closed at the same time when dataInputStream close.